### PR TITLE
185106857 Feat: admin disable account 

### PR DESCRIPTION
**Pull request by nkurunziza1**
**What does this PR do?**
- As admin  must disable the user different user account for different reason
**Checklist**
- [x] Admin disable account of users
- [x] Admin activate account of users
- [x] As admin list all user account of user and numbers

**How this should be manually testing**

- clone  this [repo](https://github.com/atlp-rwanda/destructors-ec-fe.git) 
- got to the repo directory by  writing this `cd  destructors-ec-fe` in terminal
- write this command in terminal `git checkout ft-admin-disable-account-185106857
- Run `npm run div` for copy  `http://localhost:5173/` and then paste in  browser  add `/users`
**Related pivottracker**
- admin should disable  user's account`185106857
`
Screenshot (113)
Screenshot (116)

### CODE REVIEW

**Review by JENNY-257 — APPROVED** [review]

**Review by ba-lambert — APPROVED** [review]
Perfect user interface
but the list should be not touching on the bottom
font should be poppins
1 test failing

**Review by mjeanbosco19 — APPROVED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -8,18 +8,6 @@ module.exports = {
...
-        hover: '#95CAEC',
-        hover2: '#f3f7fc',
-        background: '#fafafa',
-        red: '#de3929',
-        sidebar: '#F4F4F4',
-        active: '#E2E6EB',
-        hovercancel: '#9B1A1B',
-        cancel: '#9B1A1A',
-      },
       fontFamily: {
```
Comment (katros1) [inline tailwind.config.js:11]: why did you remove  these  colors ??

File: tailwind.config.js [hunk 2]
```diff
@@ -42,18 +30,31 @@ module.exports = {
...
       },
-    },
-    screens: {
-      xs: { max: "425px" },
-      sm: { min: "426px" },
-      md: { min: "769px" },
-      mdl:{ min: "1024px"},
-      lg: { min: "1440px" },
-      xl: { min: "2560px" },
-      laptop: { max: "1165px"}
```
Comment (katros1) [inline tailwind.config.js:54]: why did you remove these tailwind screen configuration, some of us used them in responsiveness??

**Review by katros1 — CHANGES_REQUESTED** [review]

**Comment by katros1** [comment]
also your mobile version don,t meet what we have on our figma, try to remove the status button  on mobile devices. use ellipsis dots instead as what is in figma

File: src/components/Admin/Users.jsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                    ? 'Verified' : 'UnVerified'}</pan>
+                  <span className='hidden xs:block'>{user.role}</span>
+                  <button
+                    className={`px-4 py-2 rounded-lg w-2  ${
+                      user.isActive ? 'bg-customBlue text-white' : 'bg-yellow-500 text-white'
+                    } hidden xs:block w-full xs:w-40`}
+                    onClick={() => handleConfirmation(user)}
+                    // disabled={!user.isActive}
+                    // style={{ width: '100%' }}
+
```
Comment (leandreAlly) [inline src/components/Admin/Users.jsx:51]: clean this if you don't need them

**Review by leandreAlly — APPROVED** [review]

**Pull request merged**